lib/everypolitician/popolo/entity.rb

```diff
@@ -26,6 +26,7 @@ def key?(key)
       end
 
       def ==(other)
+        return false unless other.instance_of?(self.class) 
```

Comment:
@tmtmtmtm No reason other than it was done that way in a book I was reading last night. I've refactored.